exercises/faq-page/README.md

```diff
@@ -0,0 +1,112 @@
+# Frequently Asked Questions and Cheat Sheet
+
+> This section contains common information needed when working through the exercises and a command line cheat sheet to help with completing exercises.
+
+---
+
+## Frequently Asked Questions
+
+- How can I get into my CodeReady Workspace (assumes you have gone through Exercise 1, Part 1 where we created the cloud IDE)?
+
+  1. Navigate to the CodeReady workspaces link provided by your facilitators. Take note that this is not the exact same link as in Exercise 1, Part 1 which was used to deploy the cloud IDE. After navigating to the link, you should see a page similar to the picture below.
```

Comment:
BOLD THIS - this mistake happens a lot - even when mentioned....

"Take note that this is not the exact same link as in Exercise 1, Part 1 which was used to deploy the cloud IDE."